examples/carplay-web-app/src/useCarplayAudio.ts

```diff
@@ -40,14 +40,6 @@ const useCarplayAudio = (worker: CarPlayWorker) => {
         player.feed(data)
       } else if (audio.command) {
         switch (audio.command) {
-          case AudioCommand.AudioSiriStart:
```

Comment:
Instead of a switch case the dongle sends specific commands for mic start/stop - mapped these and used instead.